test/ParseableInterface/modifiers.swift

```diff
@@ -97,3 +97,8 @@ public struct MyStruct {
   }
   // CHECK-NEXT: }
 }
+
+// CHECK-LABEL: sealed public protocol MySealedProtocol {
+sealed public protocol MySealedProtocol {
+  // CHECK-NEXT: }
+}
```

Comment:
Small formatting nit: can you add a newline on all the tests?

Reply:
Will do thanks.